src/components/modals/AddAccounts/steps/03-step-import.js

```diff
@@ -353,25 +368,25 @@ export const StepImportFooter = ({
     <Fragment>
       {currency && <CurrencyBadge mr="auto" currency={currency} />}
       {scanStatus === 'error' && (
-        <Button mr={2} onClick={() => setScanStatus('scanning')}>
-          {t('app:common.retry')}
-        </Button>
+        <Fragment>
+          <ExternalLinkButton mr={2} label={t('app:common.getSupport')} url={urls.faq} />
+          <RetryButton primary onClick={setScanStatus('scanning')} />
```

Comment:
it looks like you're accidentally calling setScanStatus instead of giving a function!